fundingmanager.go

```diff
@@ -1241,11 +1230,10 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 	}
 
 	// At this point we have received the last funding message from the
-	// initiating peer before the funding transaction will be broadcast so
-	// set the timestamp to the zero value. This indicates to the zombie
-	// sweeper that this reservation should not be pruned, as it is now
-	// waiting for the funding transaction
-	resCtx.lastUpdated = time.Time{}
+	// initiating peer before the funding transaction will be broadcast.
+	// Lock the reservation so it is not pruned by the zombie
+	// sweeper, as it is now waiting for the funding transaction.
+	resCtx.lock()
```

Comment:
Do this towards the end of the method, right before launching the go routine. This to make it more clear that we only want to "lock" it from being swept in case we successfully start waiting for the funding confirmation.

Reply:
done